### DISCUSSION TITLE
Using Nuclei Engine 2.7.7 (Outdated)

### DISCUSSION THREAD

**Discussion by Archurcl4w** [projectdiscovery/nuclei#2588] [Q&A] [answered]
![nuclei _outdated](https://user-images.githubusercontent.com/61706775/189557324-5dbf2aa3-e442-4bab-9a53-bbf87b442ff2.png)
Hello Nuclei is an awesome and very good tool . I appreciate it a lot.  But recently I am  having this issue  even my version  is the latest 2.7.7 ( Outdated ) ? Nuclei Templates 8.6.9 (latest) is  also wrong . Any kind support to resolve it ?

**Comment by forgedhallpass** [comment, marked as answer]
Hello @Archurcl4w,

I cannot reproduce it locally. Could you run a `nuclei -health-check` and share the logs?
Also the output of `cat ~/.config/nuclei/.templates-config.json | jq`?

You could also try to rename the `.templates-config.json` to `.templates-config.json.bak` and let `nuclei` re-download the templates. See if that helps.

![image](https://user-images.githubusercontent.com/13679401/190384165-6c3f6908-f827-45db-a3e3-7c03bef753c6.png)
